core-ui/src/components/Predefined/Create/VirtualServices/templates.js

```diff
@@ -0,0 +1,11 @@
+export function createTemplate(namespace) {
+  return {
+    apiVersion: 'networking.istio.io/v1beta1',
+    kind: 'VirtualService',
+    metadata: {
+      name: '',
+      namespace: namespace,
+    },
+    spec: {},
```

Comment:
why don't we have anything in the spec? we should have here the minimum to be able to create the VS, name and namespace are not enough.